pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -66,7 +66,7 @@ mod tests;
 pub mod pallet {
 	use super::*;
 
-	const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
+	const STORAGE_VERSION: StorageVersion = StorageVersion::new(3);
```

Comment:
Can you please tell me why do we jump one version here?

Reply:
Dev and Demo are using storage version 2 which represents the `DomainRouters` migration in which we removed all key types except for `AxelarEvm`. So to distinguish between the Dev/Demo storage version and the current state of the repo, we need to bump another time.